### DISCUSSION TITLE
Stable releases?

### DISCUSSION THREAD

**Discussion by DSA101** [ultralytics/yolov5#6357] [Q&A] [answered]
Does yolov5 have a concept of "stable" releases or you are always expected to get the latest version from github? Asking sine during the past couple of months of development at least once latest yolo commits broke compatibility with my code that relies on yolo. I see that now my version is behind by ~80 commits. On one hand I want the latest version, but on the other hand I don't want to introduce any unnecessary breaks to the code that is working well and would rather update once every couple of month or so, preferably to some stable version rather than to a random latest build. Any guidance how to deal with code updates is appreciated.

**Comment by glenn-jocher** [comment, marked as answer]
@DSA101 master is recommended for the latest features and fixes, but you can also checkout the latest release. Latest is v6.0:
https://github.com/ultralytics/yolov5/releases/tag/v6.0

Major releases i.e. v5.0 include model updates, minor releases i.e. v5.1 do not.
